lib/fkit/help.rb

```diff
@@ -1,6 +1,7 @@
 module FKit
   HELP_MSG = <<-EOF
 Usage: fk [cmd] (subcmd|options)
+  fk \e[94mup\e[39m           -- Create a new repository on github that is tracked by fantastic-kit
```

Comment:
For the most part irrelevant to this PR:

Also for future wer should just add a map/list of pairs which is "command, desp" and then run a loop with the ANSI coloring etc. 

Also might be a good idea to pull in the custom cmds from kit.yml and print them here.